paasta_tools/marathon_tools.py

```diff
@@ -227,7 +227,7 @@ def get_autoscaling_params(self):
         default_params = {
             'method': 'default',
         }
-        return self.config_dict.get('autoscaling', default_params)
```

Comment:
Seems ok. I've never seen this kwargs based merge thing. Is there a more pythonic way to do this operations? (dict.update?)